googletest/test/googletest-listener-test.cc

```diff
@@ -47,7 +47,7 @@ using ::testing::TestPartResult;
 using ::testing::UnitTest;
 
 // Used by tests to register their events.
-std::vector<std::string>* g_events = nullptr;
+static std::vector<std::string>* g_events = nullptr;
```

Comment:
Like the other file, let's put the whole thing in an anonymous namespace.

Reply:
Can't do this. This causes a build failure.